### DISCUSSION TITLE
Controlling directly the moteur speed or steps  vi Wab interface

### DISCUSSION THREAD

**Discussion by blotfi** [luc-github/ESP3D#637] [F.A.Q] [answered]
Dear Luc
Thanks for your great interface
what is the syntax to type on a web navigator in order to move by a certain steps the motors
http://IPaddy/???
thank you

**Comment by luc-github** [comment, marked as answer]
the commands url is 
In 2.1
`/command?commandText=`
In 3.0
`/command?cmd=`

add the GCODE command at the end and be sure you encode the command if there are special chars and space

**Comment by blotfi** [comment]
thanks, that works
